src/AudioIO.cpp

```diff
@@ -2830,7 +2836,9 @@ void AudioIO::StopStream()
 
    mPlaybackTracks.clear();
    mCaptureTracks.clear();
+#ifdef HAVE_MIDI
```

Comment:
This define does not exist, should probably be `#ifdef EXPERIMENTAL_MIDI_OUT`.  Will correct in my other PR.